### ISSUE TITLE
No preprocessing for BraTS?

### ISSUE THREAD

**Issue by adarc8** [pfriedri/wdm-3d#14]
Hey, according to the paper and the repo, the only preprocessing seems to be on LIDC-IDRI dataset, but nothing mentioned about the BraTS2023.
I saw that your BraTS dataset `get_item` is basically:
1. Clip values to quantile 0.001 and 0.999
2. Norm in [0,1] by min max
3. zero pad to 256x256x256 (where original data is 240x240x155),
4. If img_size==128 (which is my case), down-sample with an average pooling by 2
5. If training then also normalize to [-1, 1]

Did I missed something?

Im asking cause im afraid Im receiving poor results..
I know u guys trained for 10 days, but i expected 3 days of training to be much better than that:

3days of training with UNet with no wavelet transform:

![Image](https://github.com/user-attachments/assets/b11cd5df-c791-4880-8c45-fca7f8316620)

3days of training with UNet with wavelet transform:

![Image](https://github.com/user-attachments/assets/af6bd9f8-8c1a-4383-b16e-ad3c6bbdfb27)

**Comment by pfriedri** [comment]
@adarc8 We preprocess the images as described in the paper: "_For training on the BraTS dataset, we use the T1-weighted
brain MR-images, clip the upper and lower 0.1 percentile intensity values, zero pad the volumes to a size of 256 × 256 × 256 and normalize them to a range of [−1, 1]. For training on the LIDC-IDRI lung CT dataset, we initially clip all values below −1000 and the upper 0.1 percentile intensity values. We then resample the images to an isotropic voxel size of 1 mm, center crop them to a size of 256 × 256 × 256, and normalize to a range of [−1, 1]. When training on images with a resolution of 128 × 128 × 128, we perform downsampling by applying average pooling._"

While we perform some offline preprocessing for LIDC-IDRI, the rest is implemented in the dataloader.
